src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js

```diff
@@ -20,6 +21,12 @@ import styles from "./ItemSettings.less";
 
 export const ItemSettings = memo(
   function ItemSettings(props) {
+    const showSiteNameInMetaTitle = useSelector(
```

Comment:
This only subscribes to a specific objects value. If the specific object is not found (not all instances may have this setting key) then the variable is a still falsy value and does not render the title